report/zuordnung.Rmd

```diff
@@ -0,0 +1,89 @@
+---
+title: "Schule-Block-Zuordnung"
+author: "idalab GmbH"
+date: '`r format(Sys.time(), "%B %d, %Y")`'
+output: pdf_document
+params:
+  map: ""
+  blocks: ""
+  schools: ""
+  school_stats: ""
+  school_blocks: "" 
+---
+
+```{r libs, include=FALSE, message=FALSE, warning=FALSE, error=FALSE}
+#' Export a Formattable as PNG, PDF, or JPEG
+#'
+#' @param f A formattable.
+#' @param file Export path with extension .png, .pdf, or .jpeg.
+#' @param width Width specification of the html widget being exported.
+#' @param height Height specification of the html widget being exported.
+#' @param background Background color specification.
+#' @param delay Time to wait before taking webshot, in seconds.
+#'
+#' @importFrom formattable as.htmlwidget
+#' @importFrom htmltools html_print
+#' @importFrom webshot webshot
+#'
+#' @export
+export_formattable <- function(f, file, width = "100%", height = NULL, 
+                               background = "white", delay = 0.2)
+{
+  w <- formattable::as.htmlwidget(f, width = width, height = height)
+  path <- htmltools::html_print(w, background = background, viewer = NULL)
+  url <- paste0("file:///", gsub("\\\\", "/", normalizePath(path)))
+  webshot::webshot(url,
+          file = file,
+          selector = ".formattable_widget",
+          delay = delay)
+}
+```
+
+## Kartendarstellung der Einzugsbereiche
+
+```{r map, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE}
+ggmap::ggmap(params$map) +
+  ggplot2::geom_polygon(mapping = aes(x = long, y = lat, group = group, fill = 'school'),
```

Comment:
Schau mal: So bekommst du die Attribute aus dem `SpatialPolygonsDataFrame` auch in den ge`ggplot2::fortify`ten bzw ge`broom::tidy`ten Dataframe.